tests/lua-scflowstats/check.sh

```diff
@@ -0,0 +1,3 @@
+#! /bin/sh
+
+exec cmp ${TEST_DIR}/output/lua-scflowstats.log ${TEST_DIR}/expected/lua-scflowstats.log
```

Comment:
Can you remove the `check.sh` scripts? We now have a `file-compare` that can be put in `test.yaml`, see README.md for an example.